### ISSUE TITLE
Prior specification throws error when trying to set Q_sigma

### ISSUE THREAD

**Issue by ErikRingen** [ScottClaessens/coevolve#73]
When a user tries to specify a prior for Q_sigma, they get an error:

`Error: Argument 'prior' list contains names that are not allowed. Please use only the following names: 'b', 'eta_anc', 'A_offdiag', 'A_diag', 'L_R', 'Q_sigma', 'c', 'phi', 'shape', 'sigma_dist', 'rho_dist', 'sigma_group', and 'L_group'`

Here is a reproducible example:

```r
fit <- coev_fit(
  data = authority$data,
  variables = list(
    political_authority = "ordered_logistic",
    religious_authority = "ordered_logistic"
  ),
  id = "language",
  tree = authority$phylogeny,
  prior = list(
    Q_sigma = "normal(1, 1)"
  ),
  # additional arguments for cmdstanr::sample()
  chains = 4,
  parallel_chains = 4,
  seed = 1
)
```

p.s. on the topic of prior specification, it seems like right now one can only set a common prior for all traits? E.g., Q_sigma = "normal(1, 1)" specifies the same prior for all variables, without an option to set trait-specific priors?

**Comment by ScottClaessens** [comment]
Thanks for spotting! I will fix this soon.

Yes, currently it is only possible to set common priors for all traits. Perhaps we could open another issue and think about how to implement trait-specific priors?

**Comment by ScottClaessens** [comment]
Fixed in c794527.
